pkg/fftm/policyloop.go

```diff
@@ -286,22 +286,26 @@ func (m *manager) execPolicy(ctx context.Context, pending *pendingState, syncDel
 }
 
 func (m *manager) sendWSReply(mtx *apitypes.ManagedTX) {
```

Comment:
Think this function needs a name change, as it's a conditional processing.
Maybe `checkSendWSReply` or `sendWSReplyIfResolved`

Reply:
Yes agreed. Because I'm not a fan of `do-a-thing-if-a-condition-is-met` function names I've chosen to put the status check outside the `sendWSReply` function call.